lib/index.js

```diff
@@ -1 +1,121 @@
-// Your code will go here.
+"use strict";
+
+import md5 from "md5";
+import uuid from "uuid";
+import "isomorphic-fetch";
+
+var localStorage;
+if (typeof (localStorage) == "undefined" || localStorage === null) {
+  var LocalStorage = require('node-localstorage').LocalStorage;
+  localStorage = new LocalStorage('./lib/localStorage');
```

Comment:
The ``./lib/localStorage`` path seems a bit strange to me.
Should we change it with ``/tmp/tests.localStorage``?